core/integration/module_test.go

```diff
@@ -88,6 +88,25 @@ func (ModuleSuite) TestGoInit(ctx context.Context, t *testctx.T) {
 		require.Contains(t, generated, "module dagger/my-module")
 	})
 
+	t.Run("issue-7941, single character in kebab case module name", func(ctx context.Context, t *testctx.T) {
```

Comment:
nit: the issue number can be a comment here, something like `dagger/dagger#7941` - but shouldn't be in the test name.